wolfssl/wolfcrypt/cpuid.h

```diff
@@ -54,6 +54,9 @@
 
     void cpuid_set_flags(void);
     word32 cpuid_get_flags(void);
+    void cpuid_select_flags(word32 flags);
```

Comment:
We need to set WOLFSSL_API for visibility on public APIs.

Reply:
Fixed